### ISSUE TITLE
Won't Run: Numpy Error Message

### ISSUE THREAD

**Issue by matsushiro2011** [deeppomf/DeepCreamPy#118]
Error message: ""Error:  No module named 'numpy'
Could not import modules. Make sure all dependencies are installed."" Error message given immediately by IDLE interpeter when I attempt to run the script.

This error message occurs when I attempt to run the script normally following the instructions you provided.

**Installation method**
I downloaded the program and followed the instructions provided. I ran the code via the IDLE interpreter following instructions from the web on how to run a PY file. 

 - OS: Windows 10
 - Version 1803
 - Supports AVX? Don't know 

Additional context: I have almost 0 technical knowledge and am just following instructions. I probably did something wrong but can't find what it is.

**Comment by Kadantte** [comment]
You did not installed [requirements.txt](https://github.com/deeppomf/DeepCreamPy/blob/master/requirements.txt), Please check here to know how to install DCP from source or run the Binary build: 
#1 [INSTALLATION.md](https://github.com/deeppomf/DeepCreamPy/blob/master/docs/INSTALLATION.md).
#2 [INSTALLATION_BINARY.md](https://github.com/deeppomf/DeepCreamPy/blob/master/docs/INSTALLATION_BINARY.md).
For more details please check [Docs](https://github.com/deeppomf/DeepCreamPy/tree/master/docs).

**Comment by matsushiro2011** [comment]
I followed the instructions on the prebuilt binary and downloaded the latest release. Requirements.txt looks like complete gibberish to me because I am a non-coder with no technical knowledge. I may need you to **ELI5** for me step-by-step because I already attempted to follow the instructions as best I could with my limited understanding. Sorry about that! 

I actually already downloaded the prebuilt binary and attempted to run it prior to opening this issue. **Didn't work.** I also attempted to install the C++ Redistributable and the installer told me my computer already has it.

It would also be nice if you guys would make a version of this tool that didn't require any knowledge of python (which I don't have knowledge of).

But for now, please have some patience with me and bear in mind you're talking to a person with zero computer technical knowledge. I've never dabbled in anything related to coding. Thank you for your patience!

**Comment by Kadantte** [comment]
Okay then I already answer how you can install Requirements.txt and everything else as good as I can check this issue it will help since you're new with python and pip #115
